src/apis/spawnHelpers.jsx

```diff
@@ -91,7 +91,7 @@ export const spawnRenameItem = ({ selected, name, path, Dialogs, setErrorMessage
 
 export const spawnCreateDirectory = ({ name, currentPath, selected, Dialogs, setErrorMessage }) => {
     let path;
-    if (selected.icons_cnt || selected.type === "directory") {
+    if (selected.icons_cnt || selected.type === "dir") {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-210-20240129-100325-cbe55367-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)